uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-popup-files-tab.js

```diff
@@ -70,8 +75,10 @@ define([
       var filesWithSoftDelete = _.filter(vm.request.files, function (file) {
         return file.toBeDeleted;
       });
+      var queueLength = (vm.fileUploader && vm.fileUploader.queue)
```

Comment:
put this two counts into separate functions `filesMarkedForDeletion` and `fileUploaderQueue`, makes for better and more readable code
```js
vm.getFilesCount = function () {
  var filesToDelete = filesMarkedForDeletion();
  var queue = fileUploaderQueue();

  return vm.request.files.length + queue.length - filesToDelete.length; 
}